gpt_bpe_test.go

```diff
@@ -817,15 +871,3 @@ func TestModelDownloadPythiaSharded(t *testing.T) {
 	fmt.Println("All Exists - Looks good.")
 
 }
-
-func TestFunctionForShards(t *testing.T) {
```

Comment:
why is this deleted?

Reply:
was a testing function i didn't remove, its incorporated into the TestModelDownloadPythiaSharded test case